docs/features/role-based-access-control/managing-roles/brief.md

```diff
@@ -0,0 +1,59 @@
+# Managing user roles - Brief
+
+This document complements the [main documentation for Role-based access
+control](../brief.md), adding details about the workflow for assigning roles to
+users within a Marxan Cloud instance.
+
+The initial implementation is outlined here. Future reviews of this feature
+may add the ability to invite users who don't yet have an account on the
+instance, to add a role invitation step (which the grantee may accept or
+reject), to restrict the list of users that can be discovered to those within
+the parent organization/team of a project, etc.
+
+These possible features would improve the ergonomics of the workflow for adding
+users to projects or scenarios, but they also imply substantive additional
+complexity which should be avoided for a first, functional implementation.
+
+## Adding users to a project or a scenario - workflow
+
+- Project/scenario owners can click on a `+` (or similar) button next to the
```

Comment:
I understood that we'll always be adding users to a project, not to a scenario.

Reply:
you are right - I have removed these references (but also added a note about scenario-level role granting as a possible future development, if desired)